command.c

```diff
@@ -140,6 +140,8 @@ static const struct cmd_action_map action_map[] = {
 static const struct cmd_map map[] = {
    { "FAST_FORWARD",           RARCH_FAST_FORWARD_KEY },
    { "FAST_FORWARD_HOLD",      RARCH_FAST_FORWARD_HOLD_KEY },
+   { "SLOWMOTION",             RARCH_SLOWMOTION_KEY },
+   { "SLOWMOTION_HOLD",        RARCH_SLOWMOTION_HOLD_KEY },
```

Comment:
While I really don't care either way, should the constant be named `SLOW_MOTION` and `SLOW_MOTION_HOLD` to match the naming convention of the other constants?